core/controllers/skill_editor_test.py

```diff
@@ -315,8 +315,10 @@ def test_editable_skill_handler_put_fails(self):
             expected_status_int=400)
         # Check PUT returns 404 when cannot get skill by id.
         self.delete_skill_model_and_memcache(self.admin_id, self.skill_id)
+        self.put_payload['version'] = 1
         self.put_json(
-            self.url, {}, csrf_token=csrf_token, expected_status_int=404)
+            self.url, self.put_payload,
+            csrf_token=csrf_token, expected_status_int=404)
```

Comment:
Made the payload valid again in line 318 and passed them in params because the validation was raising a 400 error.

Reply:
You can do 
```suggestion
            csrf_token=csrf_token, expected_status_int=400)
```

and don't need to add anything here.

Reply:
Yeah, this is correct, but we are supposed to be checking that a 404 is returned when given skill_id is not in the Database. If I don't add the correct payload then we wouldn't be able to do that, right?

Reply:
Ah, I see, you're right!